package.json

```diff
@@ -78,7 +85,8 @@
     "plugins": [
       "@oclif/plugin-help",
       "@fmvilas/oclif-plugin-spaced-commands"
-    ]
+    ],
+    "helpClass": "./src/help/index.ts"
```

Comment:
That's not going to work. See my comment in the previous PR: https://github.com/asyncapi/cli/pull/103#issuecomment-958718524

Reply:
Hey, I checked everything, and I think that this PR is alright it is working as intended. Let me explain, so I guess there is something is weird with the previous branch. What was happening is that the upstream branch has moved forward and so did that branch, So if I pull the changes from the upstream branch it should add the commits that have been made since I created the initial branch. But this was not happening and the upstream branch was getting replaced with the branch that I created resulting in the deletion of the `studio` code. 

So in a attempt to fix this I update my Forked main branch with the `upstream/main` branch so now they are equal and when I pull the `new-help-string` branch then it is doing the same thing and replacing my master with that `new-help-string` branch. 

So this is a new branch from the `upsteam/master` and then I manually added the changes. I checked by merging this branch with the master and it was working as intended. I ran all the tests they are working have all the `studio` code and the help string is updated.

Reply:
Oh, I get it, my bad 😄

Reply:
No problem :)